internal/lib/lib.go

```diff
@@ -12,6 +12,13 @@ import (
 	"os"
 
 	_ "embed"
+
+	// Do not remove the following imports which allow supporting package
+	// vendoring by properly copying all underlying libraries
+	_ "github.com/DataDog/go-libddwaf/v2/internal/lib/darwin-amd64"
+	_ "github.com/DataDog/go-libddwaf/v2/internal/lib/darwin-arm64"
+	_ "github.com/DataDog/go-libddwaf/v2/internal/lib/linux-amd64"
+	_ "github.com/DataDog/go-libddwaf/v2/internal/lib/linux-arm64"
```

Comment:
These files are gated by a specific build tag... do these still work despite this?

Reply:
I'm especially worried by the fact all of them will be embedded if someone imports this package in go-libddwaf.
Are we avoiding it right now?

Reply:
Indeed this is the case, I guess we need to remove it 🤷

Reply:
oh you just copy/pasted old vendoring code? because it wasn't using go:embed, that's why we didn't have the issue.
Vendoring just needs a .go file to be present in the package tree, that's why the vendor.go files were enough along with the .a files.